pkg/controller/gittrack/handler_test.go

```diff
@@ -374,6 +394,16 @@ var _ = Describe("Handler Suite", func() {
 			})
 		}
 
+		var AssertNamespacedChildNameWithColon = func() {
+			BeforeEach(func() {
+				gto = testutils.ExampleGitTrackObject.DeepCopy()
+				gto.SetName("deployment-nginx-name-what-got-a-colon-in-it")
```

Comment:
```suggestion
				gto.SetName("deployment-nginx-name-that-got-a-colon-in-it")
```

Reply:
```suggestion
				gto.SetName("deployment-nginx-name-that-has-a-colon-in-it")
```
If we are going for better grammar 😉

Reply:
This was my attempt at being funny

The name is right for the test, do you still want me to go change it?